### PR TITLE
Creates EIP-2711: Separate gas payer from msg.sender

### PR DESCRIPTION
**Pull request by MicahZoltu**

### CODE REVIEW

File: EIPS/eip-2711.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Rationale
+### EIP-155 ChainID
+I'm open to discussion on whether we should enforce EIP-155 chain IDs or not.  It feels like we might as well since we are creating a new transaction type here.
+### Transaction Versioning
+It would be *better* if we developed a mechanism for enveloping arbitrary new transaction types rather than just relying on "if it has 13 items then it is of this type, if it has 9 it is of that type".  However, while there has been a lot of talk about creating a transaction envelope layer that allows us to more freely add new transaction types, no action has been taken on that front and I am loathe to wait for someone to develope such a standard for it first.
```
Comment (edsonayllon) [inline EIPS/eip-2711.md:32]: The error's coming from the word "develope." Should be "develop."

Comment (MicahZoltu) [reply EIPS/eip-2711.md]: Fixed, thanks!

**Comment by Yovaniteo** [comment]
2711

**Review by albertoblue87-netizen — COMMENTED** [review]
Listo

**Pull request merged**